lib/schema/pluginDefinitionSchema2.json

```diff
@@ -0,0 +1,289 @@
+{
+    $schema: "http://json-schema.org/schema#",
```

Comment:
Everything in this file is missing double-quotes around the attributes... this can't be valid JSON?